lab-regan/app/service/map-service.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const angular = require('angular');
+const sewerRat = angular.module('sewerRat');
+
+sewerRat.factory('mapService', ['$log', mapService]);
+
+function mapService($log){
+  $log.debug('map service');
+
+  let service = {};
+
+  service.mapData = {
+      street: {
+        desc: 'You\'re on the street...don\'t lurk around up here!',
+        down: 'pub'
+      },
+      pub: {
+        desc: 'You\'ve found the local rat pub, stay for a brew if you\'d like',
+        name: 'pub',
+        north: 'gym',
+        east: 'fortuneTeller',
+        up: 'street'
+      },
+      gym: {
+        desc: 'You just strutted into the gym, time to get swole, rat breh',
+        south: 'pub',
+        east: 'dentist'
+      },
+      dentist: {
+        desc: 'You have arrived at the dentist\'s office',
+        west: 'gym',
+        north: 'pawn shop',
```

Comment:
NO!!! I'm stuck at the dentist!!  Can't move east to the pawnShop because you misspelled it!!  :-)